source/documentation/deploying_apps/deploying_docker_image.md

```diff
@@ -28,6 +28,14 @@ There are a few more specifics about Docker image support in Cloud Foundry:
 * Privileged container mode is not supported. Features depending on this will not work.
 * Only registries that use Docker Registry API V2 are supported.
 
+### Offline Docker registry
+
+An app deployed using a Docker image depends on the availability of the Docker registry that serves that image. 
+
+If Cloud Foundry schedules an instance of your app whilst the registry is offline (for example during a restart or a scale up), Cloud Foundry will not be able to pull the image from the registry. Restarts are normal part of PaaS operation, and can be triggered by events outside of your direct control.
```

Comment:
I'm not sure what "If Cloud Foundry schedules an instance of your app" means?

Reply:
I took the phrasing from the investigation spike summary written by @chrisfarms; Chris do you have any steer on what this means? I thought it was something to do with scheduling an instance of an app for deployment as part of a scale or a restart; am I completely getting that wrong?

Reply:
For instance, CF may find a need of redeploying your application for you, may this bee maintenance of the boxes the application lives in or if application is behaving incorrectly. When that need appears, CF would attempt to download and install Docker Image as it would when user's doing a manual push.

Reply:
If we describe in the opening paragraph that scaling/restarting/restaging are all dependent on registry availability then perhaps this paragraph could focus on platform restarts, something like:

> Cloud Foundry will also restart your application in circumstances outside of your direct control such as scaling, deployments, or underlying hardware failure of the platform itself. If the registry is not available when this happens then your application instance will fail to start.

Reply:
If we describe in the opening paragraph that scaling/restarting/restaging are all dependent on registry availability then perhaps this paragraph could focus on platform restarts, something like:

> Cloud Foundry will also restart your application in circumstances outside of your direct control such as scaling, deployments, or underlying hardware failure of the platform itself. If the registry is not available when this happens then your application instance will fail to start.

(reposting because I deleted it because computers)